roles/ovb/bmc/tasks/main.yml

```diff
@@ -0,0 +1,44 @@
+- name: Copy bmc_listener program to bmc node
```

Comment:
I think this can be passed to VMs in [user-data](http://docs.openstack.org/user-guide/cli_provide_user_data_to_instances.html), instead of ansible task that is done later.